pkg/adapter/annotation/annotation.go

```diff
@@ -44,6 +44,8 @@ const (
 	// and converted image in harbor.
 	// Example value: `sha256:2d64e20e048640ecb619b82a26c168b7649a173d4ad6cf2af3feda9b64fe6fb8`.
 	AnnotationAccelerationSourceDigest = "io.goharbor.artifact.v1alpha1.acceleration.source.digest"
+	// AnnotationNydusFlag is used to indicate a image which is nydus format.
+	AnnotationNydusFlag = "containerd.io/snapshot/nydus"
```

Comment:
The `annotation.go` file is driver independent, can we move this into the driver?